### PR TITLE
update to rasa 1.9

### PR DESCRIPTION
**Pull request by akelad**
check results on current pipeline

### CODE REVIEW

**Comment by akelad** [comment]
@melindaloubser1 seems like the build for evaluation fails because the script isn't compatible with 1.8 anymore... do you have time to look into this at some point?

**Comment by melindaloubser1** [comment]
@akelad the problem with the evaluation format script is that in `intent_report.json` there is now a new entry `accuracy:0.79..` that was not there before. This is actually due to a change in `scikit-learn` (0.20.2 vs 0.22.2).

**Comment by akelad** [comment]
@melindaloubser1 it didn't post to the PR though... something seems to have gone wrong during that stage. Can you take a look?

**Comment by melindaloubser1** [comment]
@akelad I'm seeing `GET https://api.github.com/repos/RasaHQ/rasa-demo/pulls?state=open: 401 - Bad credentials // See: https://developer.github.com/v3 (Octokit::Unauthorized)` - I'm not familiar with how github actions work, have you seen this before?

**Comment by melindaloubser1** [comment]
I'm rerunning the workflow for now to see if it repeats

**Comment by akelad** [comment]
hmm its still happening i think, i can look into it

**Comment by akelad** [comment]
ok @melindaloubser1 technically it shouldn't fail this time, but whether it posts to the PR or not is apparently a bit random sometimes.. trying to get it fixed by fixing alans fork of this "comment-on-pr" github action

**Comment by akelad** [comment]
lol um. these results are not good

**Comment by melindaloubser1** [comment]
Config updates based on testing described in [this issue](https://github.com/RasaHQ/rasa-demo/issues/460)

**Comment by akelad** [comment]
how come this is failing? 🤔 does training take too long?

**Comment by melindaloubser1** [comment]
I don't know - it was passing with previous commits, maybe something with the requirements?

**Comment by melindaloubser1** [comment]
nlu cross validation fails on fifth fold of training `DIETClassifier`; can the test as a whole take too long?

**Comment by akelad** [comment]
yep, github says it's too long. Let's just do 3 folds?

**Comment by melindaloubser1** [comment]
Sure; results might be worse then right? (since less data being used for training in each fold)

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM rasa/rasa-sdk:1.7.0
+FROM rasa/rasa-sdk:1.8.2
```
Comment (erohmensing) [inline Dockerfile]:
```suggestion
FROM rasa/rasa-sdk:1.8.1
```

Comment (erohmensing) [reply Dockerfile]: sdk 1.8.2 doesn't exist, i don't think

Comment (melindaloubser1) [reply Dockerfile]: welp that was silly of me

Comment (erohmensing) [reply Dockerfile]: honestly its a fair mistake!

**Comment by erohmensing** [comment]
Requirements update looks good. I think @akelad is the one to approve for NLU performance's sake?

**Comment by akelad** [comment]
it doesn't seem to have posted the evaluation to the PR ...

**Comment by erohmensing** [comment]
oh shit wait i might know why actually

**Comment by akelad** [comment]
ah i think i know why - it takes too long and so doesn't have the credentials to post it anymore :( i think we need to change the way that script runs. There was a thread on slack about this I think?

**Comment by melindaloubser1** [comment]
# Intent Cross-Validation Results (5 folds)
|              class               |support|f1-score|                    confused_with                    |
|----------------------------------|------:|-------:|-----------------------------------------------------|
|macro avg                         |   4801|  0.8751|N/A                                                  |
|weighted avg                      |   4801|  0.8592|N/A                                                  |
|enter_data                        |    715|  0.8941|out_of_scope(21), technical_question(6)              |
|out_of_scope                      |    354|  0.5774|enter_data(29), contact_sales(14)                    |
|how_to_get_started                |    303|  0.8797|technical_question(9), install_rasa(8)               |
|affirm                            |    225|  0.8949|enter_data(5), greet(5)                              |
|contact_sales                     |    156|  0.8349|enter_data(7), ask_faq_ee(4)                         |
|greet                             |    145|  0.8828|enter_data(9), out_of_scope(3)                       |
|signup_newsletter                 |    141|  0.9586|deny(1), enter_data(1)                               |
|technical_question                |    112|  0.6043|enter_data(8), out_of_scope(6)                       |
|ask_which_events                  |    106|  0.9668|ask_faq_community_size(1), signup_newsletter(1)      |
|ask_builder                       |    105|  0.9333|ask_whoisit(1), pipeline_recommendation(1)           |
|ask_faq_opensource_cost           |    102|  0.9231|contact_sales(4), ask_faq_ee(1)                      |
|ask_whatspossible                 |    101|  0.7714|out_of_scope(5), ask_whoisit(3)                      |
|handleinsult                      |     99|  0.8774|react_negative(1), enter_data(1)                     |
|deny                              |     98|  0.7957|enter_data(6), out_of_scope(6)                       |
|ask_faq_languages                 |     93|  0.8877|enter_data(3), ask_languagesbot(3)                   |
|install_rasa                      |     92|  0.8617|how_to_get_started(4), technical_question(4)         |
|ask_howdoing                      |     90|  0.8634|ask_whatspossible(2), react_positive(2)              |
|ask_weather                       |     78|  0.9281|out_of_scope(4), ask_howdoing(2)                     |
|ask_whatisrasa                    |     75|  0.8026|how_to_get_started(8), ask_whatspossible(2)          |
|ask_faq_voice                     |     72|  0.9324|ask_isbot(1), out_of_scope(1)                        |
|ask_whoisit                       |     62|  0.7445|out_of_scope(6), ask_whatspossible(1)                |
|ask_faq_slots                     |     62|  0.9752|technical_question(2), enter_data(1)                 |
|nlu_info                          |     60|  0.8833|enter_data(4), how_to_get_started(2)                 |
|ask_faq_differencecorenlu         |     59|  0.9748|nlu_info(1)                                          |
|ask_languagesbot                  |     57|  0.9217|ask_faq_languages(4)                                 |
|ask_restaurant                    |     55|  0.9630|out_of_scope(2), contact_sales(1)                    |
|ask_how_contribute                |     53|  0.8972|ask_whatspossible(2), technical_question(2)          |
|ask_howold                        |     53|  0.8991|out_of_scope(2), ask_whatspossible(1)                |
|ask_time                          |     51|  0.9524|out_of_scope(1)                                      |
|ask_faq_channels                  |     50|  0.8000|technical_question(7), affirm(1)                     |
|ask_faq_what_is_forum             |     50|  0.9600|ask_question_in_forum(2)                             |
|ask_faq_is_programming_required   |     49|  0.9072|how_to_get_started(2), technical_question(2)         |
|ask_wherefrom                     |     49|  0.9053|out_of_scope(3), ask_whoisit(2)                      |
|react_positive                    |     48|  0.8081|affirm(5), out_of_scope(2)                           |
|nicetomeeyou                      |     47|  0.9574|greet(1), react_positive(1)                          |
|ask_howbuilt                      |     46|  0.9474|ask_builder(1)                                       |
|switch                            |     42|  0.8750|out_of_scope(2), technical_question(2)               |
|ask_question_in_forum             |     42|  0.9535|out_of_scope(1)                                      |
|human_handoff                     |     41|  0.7654|enter_data(4), contact_sales(3)                      |
|ask_faq_community_size            |     40|  0.9487|ask_faq_channels(1), ask_how_contribute(1)           |
|telljoke                          |     40|  0.9877|N/A                                                  |
|bye                               |     39|  0.8947|greet(3), affirm(1)                                  |
|react_negative                    |     37|  0.7164|handleinsult(4), deny(3)                             |
|ask_whatismyname                  |     37|  0.9351|out_of_scope(1)                                      |
|ask_isbot                         |     37|  0.8800|ask_whoisit(3), enter_data(1)                        |
|ask_faq_tutorials                 |     37|  0.9351|nlu_generation_tool_recommendation(1)                |
|thank                             |     37|  0.9722|ask_question_in_forum(1), affirm(1)                  |
|source_code                       |     30|  0.8710|how_to_get_started(1), technical_question(1)         |
|ask_whoami                        |     26|  0.8889|ask_whoisit(1), ask_whatismyname(1)                  |
|canthelp                          |     24|  0.6087|out_of_scope(4), deny(3)                             |
|ask_faq_rasax                     |     23|  0.9048|ask_faq_differencerasarasax(2), technical_question(1)|
|ask_faq_ee                        |     22|  0.8235|ask_whatspossible(1)                                 |
|ask_why_contribute                |     21|  0.9231|ask_how_contribute(2), explain(1)                    |
|ask_faq_differencerasarasax       |     16|  0.8824|ask_which_events(1)                                  |
|pipeline_recommendation           |     16|  0.8276|out_of_scope(1), enter_data(1)                       |
|ask_faq_tutorialnlu               |     15|  0.9677|N/A                                                  |
|next_step                         |     15|  0.8889|ask_whatspossible(1), out_of_scope(1)                |
|ask_faq_python_version            |     14|  0.9333|N/A                                                  |
|ask_faq_tutorialcore              |     14|  0.9286|ask_faq_tutorials(1)                                 |
|explain                           |     12|  0.6429|out_of_scope(2), ask_whatspossible(1)                |
|nlu_generation_tool_recommendation|     11|  0.8571|how_to_get_started(1), technical_question(1)         |



# Entity Cross-Validation Results (5 folds)
|   entity   |support|f1-score|precision|recall|
|------------|------:|-------:|--------:|-----:|
|micro avg   |   1556|  0.8740|   0.9050|0.8451|
|macro avg   |   1556|  0.7855|   0.8483|0.7459|
|weighted avg|   1556|  0.8631|   0.8897|0.8451|
|product     |    418|  0.9466|   0.9388|0.9545|
|language    |    290|  0.9673|   0.9656|0.9690|
|name        |    232|  0.9191|   0.9076|0.9310|
|job_function|    177|  0.7385|   0.8108|0.6780|
|company     |    135|  0.4545|   0.5882|0.3704|
|nlu_part    |    129|  0.9170|   0.9355|0.8992|
|current_api |     91|  0.9195|   0.9639|0.8791|
|location    |     46|  0.8293|   0.9444|0.7391|
|entity      |     20|  0.2222|   0.4286|0.1500|
|user_type   |     18|  0.9412|   1.0000|0.8889|

**Comment by melindaloubser1** [comment]
Ok @akelad , Is this good to merge now?

**Pull request merged**